content-module3.md

```diff
@@ -1669,6 +1669,42 @@ export default {
 } as Options<PostgreSqlDriver>;
 
 ```
+On line 6, you'll notice that a `config` parameter is imported from a `./config.js` file, located in the same folder as `orm.config.ts`.
+We therefore must create this config file as well:
+
+- `src/config.ts`
+```ts
+import loader from '@ljobse/appsettings-loader';
+import * as fs from "node:fs";
+
+const json = await fs.promises.readFile('./config.json', 'utf8');
+
+const config = loader.applyEnvConfig(JSON.parse(json));
+
+// We need a default export here. Otherwise the imported object might be undefined.
+export default config;
+```  
+This config file loads a json and exports it so it can be loaded into `src/orm.config.ts`.
```

Comment:
This JSON file contains the configurations that are going to be re-used across the application